spec/site_prism/page_objects/sections/cases/case_details_section.rb

```diff
@@ -35,6 +35,38 @@ class CaseDetailsSection < SitePrism::Section
           end
         end
 
+        section :sar_basic_details, '.sar-basic-details' do
+          section :case_type, 'tr.case-type' do
+            element :sar_trigger, 'td .sar-trigger'
+            element :data, 'td'
+          end
+
+          section :data_subject, 'tr.data-subject' do
+            element :data, 'td'
+          end
+
+          section :data_subject_type, 'tr.data-subject-type' do
+            element :data, 'td'
+          end
+
+          section :third_party, 'tr.third-party' do
+            element :data, 'td'
+          end
+
+          section :date_received, 'tr.date-received' do
+            element :data, 'td'
+          end
+
+          section :email, 'tr.requester-email' do
```

Comment:
We probably want an option for `postal_address` here too.

Reply:
Are we never going to display the postal address?

Reply:
I thought it would make sense to have a section come up as `requster-postal` here, to make testing clearer.

Reply:
well, it's the same element - just sometimes displays email address, sometimes postal address.  I'll name it something generic, but we're talking minutiae here.